### DISCUSSION TITLE
What is the state of drones?

### DISCUSSION THREAD

**Discussion by WarMechanic** [DeltaV-Station/Delta-v#1371] [General]
I'm on university holidays for about 6 weeks now, which will make a good body of time to start working on things. I've _heard_ that DeltaV wants drones back, and whatever broke drones should be fixed. But before I start working on anything, what is the actual state on drones and what do people want?

I imagine drones to be autonomous, so I want to draw inspiration from metalgearsloth's design doc on Rogue Drones (that I can't seem to find). Basically drones would be replicating silicons that get treated as a grey antag with a win condition associated with state of the station - which gives admins breathing room. The station can choose to compete with them for resources or cooperate with them to the ends of building something new and keeping the station maintained.

**Comment by deltanedas** [comment]
drones werent broken they were just easily abusable by shitters

**Reply by Bonktrauma** [reply]
Somebody already fixed them, but canceled it when upstream didn't want them

**Reply by Bonktrauma** [reply]
I cant seem to find the post

**Reply by IamVelcroboy** [reply]
It was everything up to this commit here https://github.com/space-wizards/space-station-14/pull/25372/commits/e8ccd627142d2cc4694837694d3d497db89c6b49

from https://github.com/space-wizards/space-station-14/pull/25372
